tests/experimental.libsonnet

```diff
@@ -141,4 +101,52 @@ local volumes = import 'templates/volumes.libsonnet';
       },
     },
   },
+  // `BaseTpuVmMixin` is used to convert a 2VM target to 1VM.
+  BaseTpuVmMixin:: self.BaseTpuVmTest {
+    local config = self,
+    local cleanupHook = {
+      preStop: {
+        exec: {
+          command: [
+            'bash',
+            '/scripts/cleanup.sh',
+          ],
+        },
+      },
+    },
+
+    // Disable retries
+    jobTemplate+:: {
+      spec+: {
+        backoffLimit: 0,
+      },
+    },
+    podTemplate+:: {
+      spec+: {
+        containerMap+:: {
+          monitor: null,
+          train+: {
+            image: 'google/cloud-sdk',
+            lifecycle: cleanupHook,
```

Comment:
I think this should be part of `BaseTpuVmTest`. This deletes the TPU when the Pod is terminated in kubernetes

Reply:
Done